src/librbd/ImageCtx.cc

```diff
@@ -771,7 +771,12 @@ class SafeTimerSingleton : public SafeTimer {
         "rbd_mirroring_delete_delay", false)(
         "rbd_mirroring_replay_delay", false)(
         "rbd_skip_partial_discard", false)(
-	"rbd_qos_iops_limit", false);
+	"rbd_qos_iops_limit", false)(
```

Comment:
i think unify tab & space is better :)

Reply:
@runsisi  thanx for your comment, yes, they are aligned actually, and this is just shown not aligned in github review mode, you can pull it and read it in vi, that's all aligned and unified.